### ISSUE TITLE
NVDA Alpha 29111: serious issues reading Desktop items, File Explorer items, etc

### ISSUE THREAD

**Issue by amirsol81** [nvaccess/nvda#15390]
Summary: NVDA Alpha 29111 generates huge glitches on the Desktop, File Explorer, NVDA's own menu, tree views, etc., as it fails to automatically read the item under the cursor.
### Steps to reproduce:
1. Install NVDA Alpha 29111, the latest alpha release as of this writing. You may want to restart Windows to be on the safe side though it seems to be unnecessary.
2. Just use NVDA to navigate the Desktop, File Explorer, tree views, etc.
### Actual behavior:
NVDA fails to automatically read Desktop items, files in the Explorer, items in tree views, its own menu, etc. Oftentimes it just says, "selected," as we move around. The only way to have the item names read would be to press NVDA key+Up on each focused item.
### Expected behavior:
NVDA reads every focused item automatically.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
Alpha 29111
#### Windows version:
Windows 10, Version 22H2 (OS Build 19045.3324)
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Older alphas/betas/official releases don't exhibit this issue.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not tested

**Comment by leonardder** [comment]
Are you really sure that you tested with add-ons disabled? I've seen this happening because NVDA Remote requires the SecureDesktopNVDAObject, which has been removed in #14488.

**Comment by CyrilleB79** [comment]
@amirsol81 wrote:
> Older alphas/betas/official releases don't exhibit this issue.

Could you please give specific version numbers? More specifically for the last alpha working correctly? It would help finding the change that has caused this issue.

**Comment by amirsol81** [comment]
@leonardder is right! I don't know what caused this, but NVDA Remote, if disabled or removed, eliminates the issue.
So, with this add-on disabled/removed, can we use an alternative?

**Comment by zstanecic** [comment]
Hi,

I think that tele NVDA can fix this faster.

**Comment by amirsol81** [comment]
@zstanecic Hope so! It seems that another round of encountering incompatible add-ons has started.

**Comment by amirsol81** [comment]
So I'm going to close this issue now that we know what's generating the glitch.

**Comment by XLTechie** [comment]
> So I'm going to close this issue now that we know what's generating the glitch.

You should file one with NVDA Remote.

**Comment by XLTechie** [comment]
Perhaps I'm having a brain fart here, but shouldn't Remote--with a lastTested of 
2023.1--be listed as incompatible by that alpha?

**Comment by seanbudd** [comment]
>You should file one with NVDA Remote.

A fix is already ready to go: https://github.com/NVDARemote/NVDARemote/pull/309

>Perhaps I'm having a brain fart here, but shouldn't Remote--with a lastTested of
2023.1--be listed as incompatible by that alpha?

API breaking warnings won't appear until we merge:
https://github.com/nvaccess/nvda/pull/15367

We tend to merge this later in the release cycle so that alpha users can still use add-ons and report issues with add-ons to the developers.
Now that users can manually override breaking changes with add-ons manually we may merge this sooner.

**Comment by XLTechie** [comment]
Thanks @seanbudd. Therein lies my brainfart: I had been thinking that #15367 had 
been merged.

**Comment by leonardder** [comment]
> Now that users can manually override breaking changes with add-ons we may merge this sooner.

I would really appreciate this personally.

**Comment by amirsol81** [comment]
> > Now that users can manually override breaking changes with add-ons we may merge this sooner.
> 
> I would really appreciate this personally.

Please note that at least in the case of this particular add-on, simply overriding the installation limitation isn't enough: the add-on can run, but it severely hampers NVDA's functionality.

**Comment by josephsl** [comment]
Hi, this is one of those situations where proactive participation and responses from add-on authors are critical. Folks were told about 2024.1 being an API breaking release, and the changelog did include deprecations for developers. CC @JMDAWeb

**Comment by jmdaweb** [comment]
Hi,
I'll review the pull request created by @seanbudd for NVDA Remote and merge the changes into TeleNVDA. A new development version will be released soon.
Regards.

**Comment by jmdaweb** [comment]
Hi,
The new version is ready. Enjoy.
https://github.com/nvda-es/TeleNVDA/releases/download/2023.9.7-dev/TeleNVDA-2023.9.7-dev.nvda-addon

**Comment by amirsol81** [comment]
@jmdaweb Thanks for taking care of it so promptly!
